posts/posts-router.js

```diff
@@ -0,0 +1,150 @@
+const express = require("express");
+
+const Posts = require('../data/db.js');
+
+const router = express.Router();
+
+// ```js
+// {
+//   title: "The post title", // String, required
+//   contents: "The post contents", // String, required
+//   created_at: Mon Aug 14 2017 12:50:16 GMT-0700 (PDT) // Date, defaults to current date
+//   updated_at: Mon Aug 14 2017 12:50:16 GMT-0700 (PDT) // Date, defaults to current date
+// }
+
+// | POST   | /api/posts     | Creates a post using the information sent inside the `request body`.                                                                                                        |
+// | GET    | /api/posts     | Returns an array of all the post objects contained in the database.                                                                                                         |
+// | GET    | /api/posts/:id | Returns the post object with the specified id.                                                                                                                              |
+// | DELETE | /api/posts/:id | Removes the post with the specified id and returns the **deleted post object**. You may need to make additional calls to the database in order to satisfy this requirement. |
+// | PUT    | /api/posts/:id | Updates the post with the specified `id` using data from the `request body`. Returns the modified document, **NOT the original**.      
+
+// Crud
+router.post('/', async (req, res) => {
```

Comment:
great job with the post CRUD operations. I love seeing the async  await use as well.